doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -174,10 +175,14 @@ Doc. ID: UG-SS
 - [12 Audit Log](#12-audit-log)
   - [12.1 Changing the Configuration of the Audit Log](#121-changing-the-configuration-of-the-audit-log)
   - [12.2 Archiving the Audit Log](#122-archiving-the-audit-log)
-- [13 Back up and Restore](#13-back-up-and-restore)
-  - [13.1 Back up and Restore in the User Interface](#131-back-up-and-restore-in-the-user-interface)
-  - [13.2 Restore from the Command Line](#132-restore-from-the-command-line)
-  - [13.3 Automatic Backups](#133-automatic-backups)
+- [13 Backing up and Restoring Configuration](#13-backing-up-and-restoring-configuration)
```

Comment:
Currently importing and configuring additional encryption keys is not explained. The current documentation only covers how the directory for the additional keys is configured, but nothing about the actual keys. A new section where importing and configuring additional encryption keys is explained could be added.